### PR TITLE
Translate `intermediate_css_concepts/positioning`

### PR DESCRIPTION
**Pull request by alpererdogan8**
`intermediate_css_concepts` başlığı altındaki `positioning`'i çevirdim

### CODE REVIEW

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 1]
```diff
@@ -1,65 +1,72 @@
-### Introduction
+### Giriş
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:5]:
```suggestion
---
title: 'Konumlandırma'
---

### Giriş
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 2]
```diff
@@ -1,65 +1,72 @@
...
 
-By now you have had quite a bit of practice moving elements around the screen using things like margin, padding, and flexbox. These techniques have all relied on CSS's default "positioning-mode". This default positioning-mode is intuitive, and you'll continue using it for almost all of your layout needs. However, there are other methods at your disposal that can be very useful in some situations.
+Şimdiye kadar	margin (dış kenar boşluğu), padding (iç kenar boşluğu) ve flexbox gibi öğeleri kullanarak öğeleri ekran etrafında hareket ettirme konusunda epey pratik yaptınız. Bu tekniklerin tümü CSS'nin varsayılan "konumlandırma modu" tekniğine dayanmaktadır. Bu varsayılan konumlandırma modu sezgiseldir ve neredeyse tüm düzenleme ihtiyaçlarınız için kullanmaya devam edeceksiniz. Ancak elinizdeki diğer yöntemler bazı durumlarda çok yararlı olabilir.
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
Şimdiye kadar margin (dış kenar boşluğu), padding (iç kenar boşluğu) ve flexbox gibi öğeleri kullanarak öğeleri ekran etrafında hareket ettirme konusunda epey pratik yaptınız. Bu tekniklerin tümü CSS'nin varsayılan "konumlandırma modu" tekniğine dayanmaktadır. Bu varsayılan konumlandırma modu sezgiseldir ve neredeyse tüm düzenleme ihtiyaçlarınız için kullanmaya devam edeceksiniz. Ancak elinizdeki diğer yöntemler bazı durumlarda çok yararlı olabilir.
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 3]
```diff
@@ -1,65 +1,72 @@
...
+``position: absolute`` diğer öğeleri etkilemeden ekrandaki belirli bir noktaya yerleştirmenizi sağlar. Daha açık bir ifadeyle, bir öğe üzerinde mutlak konumlandırma kullanmak, o öğeyi normal belge akışından çıkarırken, bir ata öğeye göre konumlandırır. Basit bir şekilde ifade etmek gerekirse: belgenin normal akışından çıkarılan öğeler diğer öğeleri etkilemez ve diğer öğelerden de etkilenmez. Mutlak konumlandırmayı kullanmak,  ``top``, ``right``, ``bottom``, ve ``left`` özelliklerini kullanarak öğeleri ekranda herhangi bir yere konumlandırmanıza olanak tanır. Bu özellik, diğer öğelerin hiçbirini rahatsız etmeden bir şeyi ekranda tam bir noktaya konumlandırmak istediğinizde gerçekten kullanışlıdır. Mutlak konumlandırma için birkaç iyi kullanım durumu vardır:
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
``position: absolute`` bir şeyi diğer öğeleri etkilemeden ekrandaki belirli bir noktaya yerleştirmenizi sağlar. Daha açık bir ifadeyle, bir öğe üzerinde mutlak konumlandırma kullanmak, o öğeyi normal belge akışından çıkarırken, bir ata öğeye göre konumlandırır. Basit bir şekilde ifade etmek gerekirse: belgenin normal akışından çıkarılan öğeler diğer öğeleri etkilemez ve diğer öğelerden de etkilenmez. Mutlak konumlandırmayı kullanmak,  ``top``, ``right``, ``bottom``, ve ``left`` özelliklerini kullanarak öğeleri ekranda herhangi bir yere konumlandırmanıza olanak tanır. Bu özellik, diğer öğelerin hiçbirini rahatsız etmeden bir şeyi ekranda tam bir noktaya konumlandırmak istediğinizde gerçekten kullanışlıdır. Mutlak konumlandırma için birkaç iyi kullanım durumu vardır:
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 4]
```diff
@@ -1,65 +1,72 @@
...
 
-In the following example, we are using absolute positioning to display text over an image.
 
 <p class="codepen" data-height="300" data-theme-id="dark" data-default-tab="css,result" data-slug-hash="poWyWeJ" data-editable="true" data-user="TheOdinProjectExamples" style={{"height":"300px","boxSizing":"border-box","display":"flex","alignItems":"center","justifyContent":"center","border":"2px solid","margin":"1em 0","padding":"1em"}}>
-  <span>See the Pen <a href="https://codepen.io/TheOdinProjectExamples/pen/poWyWeJ">
-  Absolute Position | CSS Positioning</a> by TheOdinProject (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>)
-  on <a href="https://codepen.io">CodePen</a>.</span>
+ TheOdinProject tarafından yazılan Pen'e bakın <span> <a href="https://codepen.io/TheOdinProjectExamples/pen/poWyWeJ">
+   Absolute Position | CSS Positioning</a> (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>)
+ <a href="https://codepen.io">CodePen</a>.</span>
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
 <span>TheOdinProject(<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>)
     tarafından (<a href="https://codepen.io">CodePen'de</a>) hazırlanan <a 
     href="https://codepen.io/TheOdinProjectExamples/pen/poWyWeJ">Absolute Position | CSS Positioning</a>  adlı ingilizce örneğe bakınız.</span>
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 5]
```diff
@@ -1,65 +1,72 @@
...
 
-### Sticky positioning
+### Yapışkan konumlandırma
 
-Sticky elements will act like normal elements until you scroll past them, then they start behaving like fixed elements. They are also not taken out of the normal flow of the document. It might sound confusing, but checking out the behavior of [this](https://codepen.io/theanam/pen/MPLBYy) example might clear things up for you. It's useful for things like section-headings. Remember being able to still see what category you're looking at while scrolling through a shop? This is how it's done!
+Yapışkan öğeler, siz onları kaydırana kadar normal öğeler gibi davranır, daha sonra sabit öğeler gibi davranmaya başlarlar. Ayrıca belgenin normal akışından da çıkarılmazlar. Kulağa kafa karıştırıcı gelebilir, ancak [bu codepen](https://codepen.io/theanam/pen/MPLBYy) örneği sizin için bazı şeyleri açıklığa kavuşturabilir. Bölüm başlıkları gibi şeyler için kullanışlıdır. Mağaza sayfasında kaydırma yaparken hâlâ hangi kategoriyi incelediğinizi görebilmeyi hatırlıyor musunuz? İşte böyle yapılır!
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
Yapışkan öğeler, siz onları kaydırana kadar normal öğeler gibi davranır, daha sonra sabit öğeler gibi davranmaya başlarlar. Ayrıca belgenin normal akışından da çıkarılmazlar. Kulağa kafa karıştırıcı gelebilir, ancak [bu ingilizce codepen](https://codepen.io/theanam/pen/MPLBYy) örneği sizin için bazı şeyleri açıklığa kavuşturabilir. Bölüm başlıkları gibi şeyler için kullanışlıdır. Bir mağaza sayfasında kaydırma yaparken hâlâ hangi kategoriyi incelediğinizi görebilmeyi hatırlıyor musunuz? İşte böyle yapılır!
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 6]
```diff
@@ -1,65 +1,72 @@
...
-3. [This CSS Tricks](https://css-tricks.com/absolute-relative-fixed-positioining-how-do-they-differ/) page should give you a different insight on the topic. You should read it as well.
-4. Finally, [this article](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/) discusses the difference between fixed and sticky positioning, it's a great read to understand the difference better.
+1. [Bu](https://www.youtube.com/watch?v=jx5jmI0UlXU) Youtube videosu hızlı bir tempodadır ancak farklı konumlandırma davranışlarının iyi bir görsel temsilini sunuyor. Hadi izleyelim.
+2. [Bu ingilizce MDN makalesi](https://developer.mozilla.org/en-US/docs/Web/CSS/position) konumlandırma ile ilgili tüm kavramsal ayrıntıları kapsar.
+3. [Bu ingilizce CSS Tricks](https://css-tricks.com/absolute-relative-fixed-positioining-how-do-they-differ/) sayfası size konu hakkında farklı bir fikir verecektir. Bunu siz de okumalısınız.
+4. Son olarak, [bu ingilizce makale](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/) sabit ve yapışkan konumlandırma arasındaki farkı tartışıyor, farkı daha iyi anlamak için harika bir okuma.
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
1. [Bu](https://www.youtube.com/watch?v=jx5jmI0UlXU)  ingilizce Youtube videosu hızlı bir tempodadır ancak farklı konumlandırma davranışlarının iyi bir görsel temsilini sunuyor. Hadi izleyelim.
2. [Bu ingilizce MDN makalesi](https://developer.mozilla.org/en-US/docs/Web/CSS/position) konumlandırma ile ilgili tüm kavramsal ayrıntıları kapsar.
3. [Bu ingilizce CSS Tricks](https://css-tricks.com/absolute-relative-fixed-positioining-how-do-they-differ/) sayfası size konu hakkında farklı bir fikir verecektir. Bunu siz de okumalısınız.
4. Son olarak, [bu ingilizce makale](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/) sabit ve yapışkan konumlandırma arasındaki farkı tartışıyor, farkı daha iyi anlamak için harika bir okuma.
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 7]
```diff
@@ -1,65 +1,72 @@
...
 </div>
 
-### Knowledge check
+### Bilgi ölçme
 
-This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+Bu bölüm, bu dersi anladığınızı kontrol etmeniz için sorular içermektedir. Aşağıdaki soruları kendi başınıza cevaplamakta zorlanıyorsanız, cevabı bulmak için yukarıdaki materyali gözden geçirin.
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
Bu bölüm, bu dersi anladığınızı kontrol etmeniz için sorular içermektedir. Aşağıdaki soruları kendi kendinize yanıtlamakta zorlanıyorsanız, yanıtı bulmak için yukarıdaki materyali gözden geçirin.
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 8]
```diff
@@ -1,65 +1,72 @@
...
-- [What is the difference between fixed and sticky positioning?](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/)
+- [Giriş](#giriş)
+- [Öğrenme çıktıları](#öğrenme-çıktıları)
+- [Statik ve göreli konumlandırma](#statik-ve-göreli-konumlandırma)
+- [Mutlak konumlandırma](#mutlak-konumlandırma)
+- [Sabit konumlandırma](#sabit-konumlandırma)
+- [Yapışkan konumlandırma](#yapışkan-konumlandırma)
+- [Ödev](#ödev)
+- [Bilgi ölçme](#bilgi-ölçme)
+- [Ek kaynaklar](#ek-kaynaklar)
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
- [Statik ve göreceli konumlandırma arasındaki fark nedir?](#static-ve-göreli-konumlandırma)
- [Mutlak konumlandırma ne işe yarar?](#mutlak-konumlandırma)
- [Sabit ve yapışkan konumlandırma arasındaki fark nedir? ilgili ingilizce makale için tıklayın.](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/)

```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 9]
```diff
@@ -1,65 +1,72 @@
...
 
-### Additional resources
+###	Ek kaynaklar
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
### Ek kaynaklar
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 10]
```diff
@@ -1,65 +1,72 @@
...
 
-This section contains helpful links to related content. It isn’t required, so consider it supplemental.
+Bu bölüm, bu dersi anladığınızı kontrol etmeniz için sorular içermektedir. Aşağıdaki soruları kendi başınıza yanıtlamakta zorlanıyorsanız, yanıtı bulmak için yukarıdaki materyali gözden geçirin.
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 11]
```diff
@@ -1,65 +1,72 @@
...
 
-- [Understand the CSS Position Property With Practical Examples](https://www.makeuseof.com/css-position-property-practical-examples/) provides some different CSS methods for positioning elements. 
+- [Understand the CSS Position Property With Practical Examples](https://www.makeuseof.com/css-position-property-practical-examples/) adlı ingilizce makale, öğeleri konumlandırmak için bazı farklı CSS yöntemleri sağlar.
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md]:
```suggestion
- [Understand the CSS Position Property With Practical Examples](https://www.makeuseof.com/css-position-property-practical-examples/) adlı ingilizce makale, öğeleri konumlandırmak için bazı farklı CSS yöntemleri sunmaktadır.
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 12]
```diff
@@ -1,65 +1,71 @@
...
+
+- kipler
+- üzerinde bir başlık bulunan resim
+- diğer öğelerin üstündeki simgeler
+
+Aşağıdaki örnekte, bir resim üzerinde metin görüntülemek için mutlak konumlandırma kullanıyoruz.
 
-In the following example, we are using absolute positioning to display text over an image.
 
 <p class="codepen" data-height="300" data-theme-id="dark" data-default-tab="css,result" data-slug-hash="poWyWeJ" data-editable="true" data-user="TheOdinProjectExamples" style={{"height":"300px","boxSizing":"border-box","display":"flex","alignItems":"center","justifyContent":"center","border":"2px solid","margin":"1em 0","padding":"1em"}}>
```
Comment (hueypov) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:30]:
```suggestion
<p class="codepen" data-height="300" data-theme-id="dark" data-default-tab="css,result" data-slug-hash="poWyWeJ" data-editable="true" data-user="TheOdinProjectExamples" style={{"height":"300px","boxSizing":"border-box","display":"flex","alignItems":"center","justifyContent":"center","border":"2px solid","margin":"1em 0","padding":"1em"}}>
```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 13]
```diff
@@ -1,65 +1,71 @@
...
 
-### Knowledge check
+### Bilgi ölçme
+
+Bu bölüm, bu dersi anladığınızı kontrol etmeniz için sorular içermektedir. Aşağıdaki soruları kendi kendinize yanıtlamakta zorlanıyorsanız, yanıtı bulmak için yukarıdaki materyali gözden geçirin.
 
-This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+- [Statik ve göreceli konumlandırma arasındaki fark nedir?](#static-ve-göreli-konumlandırma)
+- [Mutlak konumlandırma ne işe yarar?](#mutlak-konumlandırma)
+- [Sabit ve yapışkan konumlandırma arasındaki fark nedir? ilgili ingilizce makale için tıklayın.](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/)
```
Comment (huseyingulec) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:65]:
```suggestion
- [Sabit ve yapışkan konumlandırma arasındaki fark nedir? ilgili ingilizce makale için tıklayın.](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/)
```

Comment (huseyingulec) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:65]:
```suggestion
- [Statik ve göreceli konumlandırma arasındaki fark nedir?](#static-ve-göreli-konumlandırma)
- [Mutlak konumlandırma ne işe yarar?](#mutlak-konumlandırma)
- [Sabit ve yapışkan konumlandırma arasındaki fark nedir? ilgili ingilizce makale için tıklayın.](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/)
```

**Comment by hueypov** [comment]
Thank you @alpererdogan8, I have made some minor changes and fix some typos. Thank you for all your effort. 

ps. we moved the repo, please be sure to check that in kampus/monorepo.

**Pull request merged**